project/CodeCoverageSettings.scala

```diff
@@ -18,7 +18,7 @@ object CodeCoverageSettings {
 
   val settings: Seq[Setting[?]] = Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimumStmtTotal := 90,
+    ScoverageKeys.coverageMinimumStmtTotal := 80,
```

Comment:
We'd need to remember to get this back to 90

Reply:
Yeah, ill change it back after I merge this in and add the missing tests